releasenotes/notes/remove-backend-v1-et-al-cccd8f6b71a97ffc.yaml

```diff
@@ -0,0 +1,73 @@
+---
+upgrade_providers:
+  - |
+    The ``BackendV1`` model has been removed following its deprecation in 
+    Qiskit 1.2.0. This includes the ``BackendV1`` class as well as related
+    modules and utils, as they have been superseded by the :class:`.BackendV2` 
+    model. The list of removed items includes:
+
+    * ``BackendV1`` class: the core of the removed model
+
+    * All elements in `qiskit/providers/models`, as they were used to represent 
+      components of the ``BackendV1`` model:
+
+        * ``BackendConfiguration``
+        * ``BackendProperties``
+        * ``BackendStatus``
+        * ``QasmBackendConfiguration``
+        * ``PulseBackendConfiguration``
+        * ``UchannelLO``
+        * ``GateConfig`` 
+        * ``PulseDefaults`` 
+        * ``PulseQobjDef``
+        * ``Command``
+        * ``GateProperties``  
+        * ``Nduv``    
+        * ``JobStatus``: This class has been superseded by the more widely used :class:`.JobStatus`
+        * ``PulseDefaults``
+
+    * ``BackendV2Converter`` class: used to convert from ``BackendV1`` to :class:`.BackendV2` 
+
+    * ``convert_to_target`` function: used to build a :class:`.Target` instance from 
+      legacy ``BackendV1`` components (such as ``BackendConfiguration`` or ``BackendProperties``)
+
+    * ``BackendPropertyError`` and ``BackendConfigurationError``: exceptions linked to removed classes
+
```

Comment:
I think there are a couple of removals missing here. The ones I caught are:

* `Target.from_configuration()` backend_properties` arg
* `target_to_backend_properties`

I might have missed others though.